_includes/tools/sdk.html

```diff
@@ -15,10 +15,16 @@
   The ruby gem is available via <a target="_blank" href="https://rubygems.org/gems/collins_client">rubygems.org/gems/collins_client</a>.
 </p>
 <p>
-  In addition to the Ruby implementation we have a modest Bash library available that has support
-  for much of the basic core API endpoints. This is the library we use for Invisible Touch.
+  There is also a feature-complete API client written in Go. The source code is
+  available at <a target="_blank" href="https://github.com/tumblr/go-collins">github.com/tumblr/go-collins</a>.
+  The full documentation can be found on <a target="_blank" href="https://github.com/tumblr/go-collins">GoDoc</a>.
```

Comment:
Should this be a link to https://godoc.org/github.com/tumblr/go-collins/collins?